Gemfile

```diff
@@ -1,8 +1,16 @@
 raise "Ruby versions less than 2.2.2 are unsupported!" if RUBY_VERSION < "2.2.2"
+
+source 'https://rubygems.org'
+
 #
 # VMDB specific gems
 #
 
+gem "manageiq-gems-pending", ">0", :require => 'manageiq/gems/pending', :git => "https://github.com/bdunne/manageiq-gems-pending.git", :branch => "master"
+# Modified gems for gems-pending.  Setting sources here since they are git references
+gem "handsoap", "~>0.2.5", :require => false, :git => "https://github.com/ManageIQ/handsoap.git", :tag => "v0.2.5-4"
```

Comment:
This should be `:tag => "v0.2.5-5"` as of https://github.com/ManageIQ/manageiq/pull/11783, https://github.com/ManageIQ/manageiq/blob/master/gems/pending/Gemfile#L54